__tests__/.eslintrc

```diff
@@ -0,0 +1,4 @@
+---
+  extends: braintree/es6
```

Comment:
I have a strong preference for ESLint files to be JSON. more precise and just seems more… related i guess?